### ISSUE TITLE
Expose DDP connection

### ISSUE THREAD

**Issue by tvogels** [inProgress-team/react-native-meteor#15]
I believe it would be useful to expose the ddp connection in your API. In my case, I would like to listen to low-level DDP messages, and don't think I can do this with the current setup.

As a quick and dirty solution, I just added these lines to my personal copy of `Meteor.js`:

```js
// Meteor.js
  ...
  connect(endpoint) {
    Data.ddp = new DDP({
      endpoint: endpoint,
      SocketConstructor: WebSocket
    });
    this.ddp = Data.ddp; // added
    ...
```

**Comment by Mokto** [comment]
I just did what you did, thanks ! Coming in next release.

Do not hesitate to make pull requests ;)

**Comment by Mokto** [comment]
Available in 1.0.0-beta3. Thanks again !

**Comment by tvogels** [comment]
Great. Thanks!
